### PR TITLE
Add Halls of Torment to the showcase

### PR DESCRIPTION
**Pull request by coppolaemilio**
image

### CODE REVIEW

File: collections/_showcase/halls-of-torment.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+linux: true
+
+
+steam: https://store.steampowered.com/app/2218750/Halls_of_Torment/
+
+
+featured_in_home: true
+---
+
+Halls of Torment is a horde survival game in a pre rendered retro look reminiscent of RPGs from the late 90s. Choose one of many hero characters and descend into the deadly Halls of Torment. Fight unholy horrors from beyond and survive wave after wave until you face one of the tormented lords. Advance your hero with character traits, abilities, and items. Create a new powerful build during each run. Explore various underground expanses and find new powerful items that enable you to venture even deeper into the abyss.
```
Comment (JohnVeness) [inline collections/_showcase/halls-of-torment.md:29]:
```suggestion
Halls of Torment is a horde survival game in a pre-rendered retro look reminiscent of RPGs from the late 90s. Choose one of many hero characters and descend into the deadly Halls of Torment. Fight unholy horrors from beyond and survive wave after wave until you face one of the tormented lords. Advance your hero with character traits, abilities, and items. Create a new powerful build during each run. Explore various underground expanses and find new powerful items that enable you to venture even deeper into the abyss.
```

File: collections/_showcase/halls-of-torment.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Features
+
+- Quick and casual runs of maximum 30 minutes
+- Old school pre rendered artstyle
```
Comment (JohnVeness) [inline collections/_showcase/halls-of-torment.md:34]:
```suggestion
- Old-school pre-rendered artstyle
```

File: collections/_showcase/halls-of-torment.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Quest based meta progression
```
Comment (JohnVeness) [inline collections/_showcase/halls-of-torment.md:35]:
```suggestion
- Quest-based meta progression
```

File: collections/_showcase/halls-of-torment.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- Large selection of diverse abilities, traits, and items, all enabling you to create interesting synergies
+- Diverse bosses featuring unique mechanics and attack patterns
+- Items can be retrieved to the overworld and used to customize future runs
+- Dozens of interesting characters allowing many different playstyles
+- Unlock and explore multiple interesting and challenging underground worlds
+- Interactable environments: Use traps and hazards to your advantage!
+
+## Early Access Content
+
+- 3+ stages unique environment
```
Comment (JohnVeness) [inline collections/_showcase/halls-of-torment.md:45]: Not sure what this means!

**Comment by coppolaemilio** [comment]
@JohnVeness I took that content from their steam page, so I don't know if I would accept your changes on their behalf 😅 Thanks for the review though!

**Pull request merged**